mixins/api.js

```diff
@@ -0,0 +1,114 @@
+import { mapState } from 'vuex'
+
+export default {
+  methods: {
+    async pollApi () {
+      // fm stream
+      await this.$axios.get('/?filter[slug]=wnyc-fm939&include=current-airing.image,current-show.show.image,current-episode.segments', { baseURL: this.$config.whatsOnAPI })
+        .then(response => (
+          this.setTheState(response.data, 0)
+        ))
+      // am stream
```

Comment:
do we need both the fm and am streams?

Reply:
I didn't touch this code at all, not sure.

But looking at it now this is probably not needed. i'll take it out.